### PR TITLE
Mes 5203 mod2 fail

### PR DESCRIPTION
**Pull request by naqvis5**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: test/e2e/features/cat-mod2/01-pass-journey.feature [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    Background:
+      Given I am logged in as "desexamineram2" and I have a test for "Dr Fox Farrell"
+      When I start the test for "Dr Fox Farrell"
+      And the candidate completes the declaration page
+      And the candidate confirms their declaration
+      Then I should see the "Declaration - Fox Farrell" page
+      When the candidate requests to receive results by post
+      And I proceed to the bike
+
+      Scenario: Examiner changes test category to A
```
Comment (turnerc1) [inline test/e2e/features/cat-mod2/01-pass-journey.feature]: Change this scenario title to something that makes sense. Like 'Driving Examiner Passes mod2 a candidate'

File: test/e2e/features/cat-mod2/02-fail-journey.feature [hunk 2]
```diff
@@ -0,0 +1,61 @@
+@catm2 @full_smoke @regression
+
+Feature: Driver Examiner complete the pass journey for Mod2
```
Comment (turnerc1) [inline test/e2e/features/cat-mod2/02-fail-journey.feature]: This feature file is for fails, change title

**Review by matthew2564 — APPROVED** [review]

**Review by turnerc1 — CHANGES_REQUESTED** [review]

File: test/e2e/features/cat-mod2/01-pass-journey.feature (hunk 1 above)
Comment (naqvis5) [reply test/e2e/features/cat-mod2/01-pass-journey.feature]: done

File: test/e2e/features/cat-mod2/02-fail-journey.feature (hunk 2 above)
Comment (naqvis5) [reply test/e2e/features/cat-mod2/02-fail-journey.feature]: done

**Review by turnerc1 — APPROVED** [review]

**Pull request merged**